Sources/MapboxCoreNavigation/Date.swift

```diff
@@ -15,6 +15,6 @@ extension Date {
     
     var nanosecondsSince1970: Double {
         // UnitDuration.nanoseconds requires iOS 13
-        return timeIntervalSince1970 * 1e6
+        return timeIntervalSince1970 * 1e9
```

Comment:
This was an old bug that hasn't affect anything, but now this line matters.

Reply:
How embarrassing! Can’t wait to use Measurement more consistently once we can require newer iOS versions.